src/index.js

```diff
@@ -0,0 +1,23 @@
+import "./database";
+import express from 'express';
+import inquiryRoutes from './routes/inquiry.route';
+// import blogRoutes from './routes/blog.route';
```

Comment:
It's not that good practice to push comments which doesn't document the code on production